AnkiDroid/src/test/java/com/ichi2/anki/RobolectricTest.java

```diff
@@ -91,4 +95,23 @@ protected JSONObject getCurrentDatabaseModelCopy(String modelName) throws JSONEx
         Models collectionModels = getCol().getModels();
         return new JSONObject(collectionModels.byName(modelName).toString().trim());
     }
+
+    protected <T extends AnkiActivity> T startActivityNormallyOpenCollectionWithIntent(Class<T> clazz, Intent i) {
+        ActivityController multimediaController = Robolectric.buildActivity(clazz, i)
+                .create().start().resume().visible();
+        //noinspection unchecked
+        return (T) multimediaController.get();
+    }
+
+    protected Note addBasicNote(String front, String back) {
```

Comment:
```suggestion
    protected Note getNoteUsingBasicModel(String front, String back) {
```
...or similar, just a bit more unambiguous that the note itself isn't basic (maybe means a simple note?) but that it is attached to the basic model